src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is started");
```

Comment:
What exactly did he start?)

Reply:
Oh, in my mind, they wound up but did not do work. Fix it and push?